README.md

```diff
@@ -34,14 +33,13 @@ For more details please refer to [versions comparison](https://github.com/gitext
   <tr>
     <td>Development</td>
     <td>MS VS 2017 v15.7+, C#7.3<br/>
-        (C++ including Visual C++ ATL for x86 and x64 for installer)
+        VC++ (inc. ATL for x86/x64 for installer)
     </td>
     <td>No official support</td>
```

Comment:
I suggest removing Mono column completely for 3.0

Reply:
done